cli/group-add.go

```diff
@@ -63,19 +63,28 @@ func GetGroupAddCmd() *cobra.Command {
 
 func groupAddCmd(cmd *cobra.Command, args []string) error {
 	groupname := args[0]
-	username := args[1]
+	usernames := args[1:]
 
 	gortClient, err := client.Connect(FlagGortProfile)
 	if err != nil {
 		return err
 	}
 
-	err = gortClient.GroupMemberAdd(groupname, username)
-	if err != nil {
-		return err
+	var errs int
+
+	for _, name := range usernames {
+		output := fmt.Sprintf("User added to %s: %s", groupname, name)
```

Comment:
Minor: Could success/failure be tracked in a bool to make the flow easier to follow?